File: Luyencodenet/Luyencode.net/DK10.cpp
#include <iostream>
using namespace std;
bool check_leap_year(int y) {
	if (y % 400 == 0 || (y % 100 != 0 && y % 4 == 0)) {
		return true;
	}
	return false;
}
int main() {
	int month, year;
	cin >> month >> year;
	if (month > 12) {
		cout << "INVALID";
	}
	else {
		switch (month) {
		case 1: case 3: case 5: case 7: case 8: case 10: case 12:
			cout << 31 << endl;
			break;
		case 4: case 6: case 9: case 11:
			cout << 30 << endl;
			break;
		case 2:
			if (check_leap_year(year)) {
				cout << 29 << endl;
			}
			else {
				cout << 28 << endl;
			}
			break;
		}
	}
	return 0;
}

File: Luyencode.net/Python/DK10.py
def check_leap_year(y):
    return y % 400 == 0 or (y % 100 != 0 and y % 4 == 0)
month, year = map(int, input().split())
if month > 12: print("INVALID")
else:
    if month in [1, 3, 5, 7, 8, 10, 12]: day = 31
    elif month in [4, 6, 9, 11]: day = 30
    elif month == 2: day = 29 if check_leap_year(year) else 28
    print(day)